src/Editor/EditorUtils.js

```diff
@@ -27,12 +27,30 @@ export const displayTextWithMentions = (inputText, formatMentionNode) => {
             });
         }else{
             formattedText.push(retLine);
-        }            
+        }
         formattedText.push("\n");
-    });    
+    });
     return formattedText;
 }
 
+export const formattedTextToPlain = (inputText) => {
+  let output = '';
+  const mentions = EU.findMentions(inputText)
+  if(mentions.length  === 0) {
+    return inputText
+  }
+  let lastIdx = 0;
+  mentions.forEach(mention => {
+    output += inputText.substring(lastIdx, mention.start)
+    output += '@' + mention.username
+    lastIdx = mention.end + 1
+  })
+  if(lastIdx < inputText.length - 1) {
+    output += inputText.substr(lastIdx)
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

```suggestion
    output += inputText.substring(lastIdx)
```

Reply:
Will change to `substring`